homework_postgres/PostgreSQL-Students.ipynb

```diff
@@ -40,9 +40,22 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.                                    FROM (SELECT movie.id, movie.title, roles.person_id
</pre><p>Why from here, multiple left join is the answer&nbsp;here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/haicheviet/dataeng-course-1/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='136'/>